sdk/servicebus/service-bus/test/utils/envVarUtils.ts

```diff
@@ -65,39 +71,69 @@ export function getEnvVars(): { [key in EnvVarKeys]: any } {
 
   return {
     [EnvVarKeys.SERVICEBUS_CONNECTION_STRING]: getEnvVarValue(
-      EnvVarKeys.SERVICEBUS_CONNECTION_STRING
+      EnvVarKeys.SERVICEBUS_CONNECTION_STRING,
+      forBrowser
     ),
-    [EnvVarKeys.QUEUE_NAME]: getEnvVarValue(EnvVarKeys.QUEUE_NAME) || "partitioned-queue",
+    [EnvVarKeys.QUEUE_NAME]:
+      getEnvVarValue(EnvVarKeys.QUEUE_NAME, forBrowser) || forBrowser
+        ? "partitioned-queue-browser"
+        : "partitioned-queue",
```

Comment:
I believe it is simpler to keep all the "browser" suffix logic in a single place. Right now, it is in both `getEnvVars` and `getEnvVarValue`. What do you think of moving the "default" values that we return here (like "partitioned-queue" and "partitioned-queue-browser") to `getEnvVarValue` ?

That way the code here would get simplified to 

```javascript
return {
  [EnvVarKeys.QUEUE_NAME]: getEnvVarValue(EnvVarKeys.QUEUE_NAME, forBrowser),
  [EnvVarKeys.QUEUE_NAME_NO_PARTITION]: getEnvVarValue(EnvVarKeys.QUEUE_NAME_NO_PARTITION, forBrowser),
  [EnvVarKeys.QUEUE_NAME_SESSION]: getEnvVarValue(EnvVarKeys.QUEUE_NAME_SESSION, forBrowser),
...
...
..

}

Reply:
So, since we defined default values, this can't really be avoided.
It's better to keep consistent with how define and use entities for Node because current changes allow users to define their own Node and Browser entities optionally, and not necessarily have to use the defaults.

So if we want to get ride of this option altogether - we can update both Node and Browser entity references to use only the defined ones, then we likely don't need those to be supplied in the .env file at all. 

Hope this makes sense! Appending browser to result would not allow for users to define or work with their own entities if they have defined in the .env file

Reply:
I didn't say that we should not define the default values at all. Neither did I say that we should not allow users to define the names. I only meant to move the code that falls back to the default value into the `getEnvVarValue()` method something like the below

```javascript
function getEnvVars(forBrowser?: boolean) {
// what you have now in the PR before returning

if (result) {
   return result;
}

  switch (name) {
    case EnvVarKeys.QUEUE_NAME:
        result = "partitioned-queue"
        break;
    case EnvVarKeys.QUEUE_NAME_NO_PARTITION
        result = "unpartitioned-queue"
    ...
    ...
  }

  if (result && isBrowser) {
    result += "-browser"
  }

return result
}
```

This way, the code in `getEnvVars()` can be simplified as described in 		https://github.com/Azure/azure-sdk-for-js/pull/3785#discussion_r293994344

Reply:
Ah okay, looks like its been clarified as to how the _BROWSER and -browser suffixes are being used and that we want to only move around the code. 

So I think, the implementation we have now is simple in terms of keeping functions short. 
The moving of code around will bloat up the `getEnvVars()` function and this file. It may help with grouping up some of the browser suffix related changes, but given that these are spread out across multiple .env, .conf, script and source files - not sure how useful this grouping would be.

My intent was to simplify the code changes to make and efforts mostly :) Moving around of code and testing of changes can be done but will just take some more time and efforts.

Reply:
Uhm, the above implementation still asks to append "-browser" to entity name if in browser, this would impede users from being able to define their own names for the browser entities, right?

Reply:
> Uhm, the above implementation still asks to append "-browser" to entity name if in browser, this would impede users from being able to define their own names for the browser entities, right?

Please observe the early exit if user defined values exist

Reply:
https://github.com/ramya-rao-a/azure-sdk-for-js/commit/eba3f4f2f5da884af0f443a8026cae5581e0aa37

Reply:
Double checked on corner cases and confirmed that the side-effects do not hold true as empty env variables continue being empty with moving of default value computation. 

Added these changes in - we could have alternatively defined a getDefaultValue() utility to house these. For now, added a comment to explain the changes.

Sorry, with the disparity in code snippet using `getEnvVars()` and comment referring to move code to `getEnvVarValues()` which is where the snippet was to live in, it was confusing what was intended - so I wasn't taking the suggestion verbatim and ignored the early check thinking it was a typo too :) 

cc @ramya-rao-a